### PR TITLE
Replaces instances of 'latn' with 'en' and updates snaps + localises indecl table. 

### PR DESCRIPTION
**Pull request by smkvr98**
Deals with substep 2 and 3 of step 2 of [localisation process](https://github.com/digitalpalitools/web-ui/issues/12). The localisation of the word "indeclinable" in indecl table was left out erroneously in PR #8 which has now been dealt with.

### CODE REVIEW

**Review by parthopdas — APPROVED** [review]

File: pls_core/src/inflections/mod.rs [hunk 1]
```diff
@@ -334,7 +334,7 @@ pub fn get_abbreviations_for_locale(
     let sql: String;
     if locale == "xx" {
         sql = "select name, description, '^' || name || '$' from _abbreviations".to_string();
```
Comment (parthopdas) [inline pls_core/src/inflections/mod.rs:336]: This doesn't sound right. Aren't we getting the latn string and adding ^$ to it. Let's discuss this please. I don't understand the syntax

**Review by parthopdas — APPROVED** [review]

File: pls_core/src/inflections/mod.rs (hunk 1 above)
Comment (smkvr98) [reply pls_core/src/inflections/mod.rs:336]: yeah, since the latn string is in the name column, I've handled the concatenation within the sql string.

Comment (parthopdas) [reply pls_core/src/inflections/mod.rs:336]: oh wow very cool!

Comment (parthopdas) [reply pls_core/src/inflections/mod.rs:336]: push through please when you are ready

**Pull request merged**